docker-compose-elastic-cloud.yml

```
@@ -0,0 +1,125 @@
+version: "2.1"
+services:
+  opbeans-node:
+    build: .
+    image: opbeans/opbeans-node:latest
+    ports:
+    - "127.0.0.1:${OPBEANS_SERVER_PORT:-8000}:3000"
+    logging:
+      driver: 'json-file'
+      options:
+        max-size: '2m'
+        max-file: '5'
+    environment:
+      - ELASTIC_APM_SERVICE_NAME=${ELASTIC_APM_SERVICE_NAME:-opbeans-node}
+      - ELASTIC_APM_SERVER_URL=${ELASTIC_APM_SERVER_URL:-http://apm-server:8200}
+      - ELASTIC_APM_APPLICATION_PACKAGES=co.elastic.apm.opbeans
+      - ELASTIC_APM_JS_SERVER_URL=${ELASTIC_APM_JS_SERVER_URL:-http://localhost:8200}
+      - ELASTIC_APM_JS_SERVICE_NAME=${ELASTIC_APM_JS_SERVICE_NAME:-opbeans-react}
+      - NODE_ENV=testing
```

It's more common to call the environment `test` than `testing` I think, right?

```suggestion
      - NODE_ENV=test
```